src/service-override/extensions.ts

```diff
@@ -50,11 +50,12 @@ import { IStorageService } from 'vs/platform/storage/common/storage'
 import { ILabelService } from 'vs/platform/label/common/label'
 import { ExtensionKind } from 'vs/platform/environment/common/environment'
 import { ExtensionDescriptionRegistrySnapshot } from 'vs/workbench/services/extensions/common/extensionDescriptionRegistry'
-import getOutputServiceOverride from './output'
 import { changeUrlDomain } from './tools/url'
 import { registerAssets } from '../assets'
 import { unsupported } from '../tools'
 import 'vs/workbench/api/browser/extensionHost.contribution'
+// We need to import missing service here because extHost.common.services registers ILoggerService and we want the implementation from missing-service to win
```

Comment:
Is there still a way in this case to use the one from `extHost`? I can imagine that's used to log logs from extensions.

Reply:
This file is not supposed to be used in the app but in the extHost worker instead. We include it by hands for the local extHost but we don't want it to override the services. The extHost logger service rôle is to send the message through the rpcProtocol to the app logger service